plugins/cue_points/quiz/services/QuizUserEntryService.php

```diff
@@ -24,20 +24,32 @@ public function submitQuizAction($id)
 			throw new KalturaAPIException(KalturaErrors::INVALID_OBJECT_ID, $id);
 
 		if ($dbUserEntry->getType() != QuizPlugin::getCoreValue('UserEntryType',QuizUserEntryType::QUIZ))
-		{
-			throw new KalturaAPIException(KalturaErrors::INVALID_OBJECT_TYPE, $dbUserEntry->getType());
-		}
+			throw new KalturaAPIException(KalturaQuizErrors::PROVIDED_ENTRY_IS_NOT_A_QUIZ, $id);
+
 		/**
 		 * @var QuizUserEntry $dbUserEntry
 		 */
 		$score = $dbUserEntry->calculateScore();
 		$dbUserEntry->setScore($score);
-//		$dbUserEntry->setStatus(QuizUserEntryStatus::QUIZ_SUBMITTED);
+		KalturaLog::debug("Quiz score is [" .$score."]");
```

Comment:
You don't need it because few lines later you save the object to the DB, that save logs the exact values that saved to the DB including the score.